nvsvocprez/utils.py

```diff
@@ -239,28 +239,47 @@ def exists_triple(s: str):
   return True if bool(int(count)) else False
 
 
+def get_ontologies() -> Dict:
+    """Get ontologies from livbodcsos ords endpoint.
+    
+        Returns (Dict): Dict of parsed ontology data. {ontology_prefix : {ontology_object}, ...}.
+    """
+    if ORDS_ENDPOINT_URL is None:
+        logging.error("Environment variable ORDS_ENDPOINT_URL is not set.")
+        return {}
+    try:
+        url = f"{ORDS_ENDPOINT_URL}/alexk/alexk/ontology"
+        resp_json = requests.get(url).json()
+        ont_data_by_url = {ont["prefix"]: ont for ont in resp_json['items']}
```

Comment:
Super minor, but variable should be ont_data_by_prefix

Reply:
The pain of copy and paste 👀

I've made all these changes now.